src/ChainSafe.Gaming.Unity/UnityOperatingSystemMediator.cs

```diff
@@ -1,14 +1,34 @@
-﻿using ChainSafe.Gaming.Web3.Environment;
+﻿using System;
+using ChainSafe.Gaming.Web3.Environment;
 using UnityEngine;
 
 namespace ChainSafe.Gaming.Unity
 {
     public class UnityOperatingSystemMediator : IOperatingSystemMediator
     {
-        public string ClipboardContent
+        public bool IsMobilePlatform => Application.isMobilePlatform;
+
+        public Platform Platform
         {
-            get => GUIUtility.systemCopyBuffer;
-            set => GUIUtility.systemCopyBuffer = value;
+            get
+            {
+                if (Application.isEditor)
+                {
+                    return Platform.Editor;
+                }
+
+                switch (Application.platform)
+                {
+                    case RuntimePlatform.IPhonePlayer:
+                        return Platform.IOS;
+                    case RuntimePlatform.Android:
+                        return Platform.Android;
+                    case RuntimePlatform.WebGLPlayer:
+                        return Platform.WebGL;
+                    default:
+                        return Platform.Desktop;
```

Comment:
You can convert to switch expression, so it's more readable

![image](https://github.com/ChainSafe/web3.unity/assets/127766834/733d5b30-bf2c-4350-9c21-852072d977a2)

Reply:
sure, will do

Reply:
done